tests/unit/h/views/groups_test.py

```diff
@@ -136,4 +145,5 @@ def test_read_noslug_redirects(pyramid_request, factories):
 def routes(pyramid_config):
     pyramid_config.add_route("group_read", "/g/{pubid}/{slug}")
     pyramid_config.add_route("api.group", "/api/group/{id}")
+    pyramid_config.add_route("api.group_members", "/api/groups/{pubid}/members")
```

Comment:
The use of `id` vs `pubid` depending on the route is unfortunate. Ideally the same param name would be used for all routes.